changes/2128.feature.rst

```diff
@@ -0,0 +1 @@
+Add `boolean_question()` API to Console.
```

Comment:
As I mentioned during our call this week, the release note should be in the voice of "advertising/marketing copy", not a commit message:
```suggestion
It is now possible to ask boolean questions using the Console interface.
```